(blog): Add D&I blog post from writing sample (#21381)
gatsbyjs/gatsby: docs/blog/2020-02-17-changing-the-oss-status-quo/index.md

> ---

Reviewer: You could add OSS to `typo-ci.yml` file for the filename, since that will likely come up again in the future.

title: "Changing the Open Source Status Quo"
date: 2020-02-17
author: Aisha Blake
excerpt: "How can we increase the diversity of open source contributors? As maintainers, we need to listen to a diverse group of community members and make thoughtful changes to create a truly inclusive project."
tags:
  - open-source
  - community
---

> In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it's often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech effectively shuts people out of the process. Black, indigenous, and people of color (BIPOC) and women are at some of the most visible of these intersections. There are many more. Consider _who_ contributes to open source.

Reviewer: ```suggestion
In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it's often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech, effectively shuts people out of the process. Black, indigenous, and people of color (BIPOC) and women are at some of the most visible of these intersections. There are many more. Consider _who_ contributes to open source.
```
Adding a comma here, let me know if you think it's misplaced. One other addition you could consider is to conclude the paragraph with a teaser of what the post will be about, i.e. "Consider _who_ contributes to open source; that's what we'll do in this post."

> The status quo is made and remade every time we as maintainers add to our documentation. Every time we, as maintainers, engage with the community via social media, issue queues, and our documentation, we help decide who contributes. We contribute to their safety and their productivity. Right now, barriers to contribution are severely limiting our capacity to create and sustain open source projects.

Reviewer: ```suggestion
The status quo is made and remade every time we as maintainers add to our documentation. Every time we, as maintainers, engage with the community via social media, issue queues, and our documentation, we help decide who contributes. We contribute to their safety and their productivity. Right now, barriers to contribution are severely limiting our industry's capacity to create and sustain open source projects.
```
Perhaps opening this up to the whole industry will reflect that this isn't unique to Gatsby (since it's OSS-wide)?

There's an opportunity here. There are thousands of highly skilled potential contributors ready, willing, and able to contribute to open source projects but they need to know that they'll be safe in doing so. A handful of projects are intentionally focusing on this work and making incredible strides towards a more inclusive open source community.

> Gatsby's most admirable quality is the company's commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is "You Belong Here". We back it up with a clear code of conduct that we consistently reference and enforce. The documentation is vast, includes specifics around contributing, and provides materials that make it easy for people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered SWAG as a thank you. There are even a limited number of one-on-one mentorship opportunities available.

Author: I think it might be useful to break these out into recommendations or maybe even call out other repos with good practices.

Reviewer: That would be cool, but I don't think it's explicitly necessary to get your point across. So up to you!

> Gatsby's most admirable quality is the company's commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is "You Belong Here". We back it up with a clear code of conduct that we consistently reference and enforce. The documentation is vast, includes specifics around contributing, and provides materials that make it easy for people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered SWAG as a thank you. There are even a limited number of one-on-one mentorship opportunities available.

Reviewer: ```suggestion
Gatsby's most admirable quality is the company's commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is "You Belong Here". We back it up with a clear code of conduct that we consistently reference and enforce. The documentation is vast, includes specifics around contributing, and provides materials that make it easy for people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered swag as a thank you. There are even a limited number of one-on-one mentorship opportunities available.
```
Should swag be in all-caps?

Author: It's short for "stuff we all get"!

Author: I'm taking the suggestion for consistency with other posts and docs but let the record show that I still think it _should_ be in all caps! 😂

Consider your own community and adopt the strategies that are feasible for your project. Who comes back to contribute more than once? Who is heard when making decisions as a group? Who feels empowered to speak publicly about your project? When interpersonal issues arise, who is believed? Take the time to examine your project's processes and documentation. What's missing? What's out-of-date? Tear down as many barriers to entry as you can identify. The more people are encouraged and prepared to participate, the more likely it becomes that you'll be able to share the burden of maintenance across a wider team.

> At the same time, listen to people in marginalized groups. Show up and shut up. Support individual contributors and affinity groups. Participate in those groups where your presence doesn't introduce a safety concern. (Ask if you have any doubts, and maybe even if you don't.) You could do this by sponsoring an event like the [Blacks in Technology Conference](https://bitcon.tech/) or attending a local Lesbians Who Tech event. Once you're there, avoid centering your own experience. Absorb as much as you can from the experiences of others and educate yourself on how to be the best ally possible. Keep in mind that you don't get to _decide_ that you are an ally. We all have to earn trust over time.

Reviewer: ```suggestion
At the same time, listen to people in marginalized groups. Show up and take a seat. Support individual contributors and affinity groups. Participate in those groups where your presence doesn't introduce a safety concern. (Ask if you have any doubts, and maybe even if you don't.) You could do this by sponsoring an event like the [Blacks in Technology Conference](https://bitcon.tech/) or attending a local Lesbians Who Tech event. Once you're there, avoid centering your own experience. Absorb as much as you can from the experiences of others and educate yourself on how to be the best ally possible. Keep in mind that you don't get to _decide_ that you are an ally. We all have to earn trust over time.
```
It's less alliterative, I know–but does "take a seat" have the same impact as "shut up"? That might align better with Gatsby's code of conduct.

Author: I take your point! I chose it less for the alliteration and more because that specific phrase is commonly used to communicate that an aspiring ally should start by listening. I'll fiddle with it but will likely change the phrase entirely rather than swap out part of it.

Here are some truly wonderful folks who are already sharing valuable resources and insights:

- [Tatiana Mac](https://twitter.com/TatianaTMac)

> - [Kim Crayton](https://twitter.com/KimCrayton1)

Reviewer: Note on the Typo CI bot here–I emailed the creator of the project to see if there's a way to opt-out of proper nouns, because we definitely don't want to add everyone's surnames to a config.

---

> title: "Changing the OSS Status Quo"

Author: Still not sure about the title... 😂😭

date: 2020-02-17
author: Aisha Blake
excerpt: "How can we increase the diversity of open source contributors? As maintainers, we need to listen to a diverse group of community members and make thoughtful changes to create a truly inclusive project."
tags:
  - open-source
  - community
---

> In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it’s often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech (BIPOC and women being some of the most visible), effectively shuts people out of the process. Consider who contributes to open source.

Author: Maybe add something here talking about demographics?

> The status quo is made and remade every time you as a maintainer add to your documentation. Every time you, as a maintainer, engage with the community via social media, issue queues, and your documentation, you help decide who contributes. You contribute to their safety and their productivity. Right now, barriers to contribution are severely limiting our capacity to create and sustain open source projects.

Author: Does it still make sense to frame this as an appeal to other maintainers? Does it still make sense if I switch to Gatsby devs, for example?

Reviewer: I think it works either way. Blog posts have a more general reach than docs do, so opening it up to other maintainers makes sense to me.

There's an opportunity here. There are thousands of highly skilled potential contributors ready, willing, and able to contribute to open source projects but they need to know that they'll be safe in doing so. A handful of projects are intentionally focusing on this work and making incredible strides towards a more inclusive open source community.

> Gatsby's most admirable quality is the company’s commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is “You Belong Here”. They back it up with a prominent, clear code of conduct that they consistently reference and enforce. The documentation is vast and includes specifics around contributing as well as providing materials that make it easy for  people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered SWAG as a thank you. There are even a limited number of one-on-one mentorship opportunities available.

Author: I feel like "we" makes more sense now that I've been hired. Also, so far, my only experience with what I saw as a code of conduct violation involved having to prod a bit to get some consensus and then eventually action. How do people feel we do with this in general?

Reviewer: Issues related to the Code of Conduct come up so rarely that we haven't had to do much–I've seen it come up once on GitHub and a comment to the person sufficed (they're still contributing, and doing a fine job).

At the same time, listen to people in marginalized groups. Show up and shut up. Support individual contributors and affinity groups. Participate in those groups where your presence doesn’t introduce a safety concern. (Ask if you have any doubts, and maybe even if you don’t.) You could do this by sponsoring an event like the Blacks in Technology Conference or attending a local Lesbians Who Tech event. Once you're there, avoid centering your own experience. Absorb as much as you can from the experiences of others and educate yourself on how to be the best ally possible. Keep in mind that you don't get to decide that you are an ally. We all have to earn trust over time.

> Here are some truly wonderful folks who are already sharing valuable resources and insights:

Author: Would a brief snippet for each person be helpful? I'm thinking links to their Twitter and/or personal sites with permission at minimum.

Reviewer: That would be awesome. Either way, though, do you think we should get permission from people before adding them to this list?

Author: Yes, definitely! I'll do that either way.

Author: Thought I'd already commented on this but I'm in the process of doing exactly this!
